platform/viewer/src/config.js

```diff
@@ -15,31 +15,33 @@ window.info = {
   homepage,
 };
 
-// For debugging
-//if (process.env.node_env === 'development') {
 window.cornerstone = cornerstone;
-window.cornerstoneWADOImageLoader = cornerstoneWADOImageLoader;
-//}
-
-cornerstoneWADOImageLoader.external.cornerstone = cornerstone;
-cornerstoneWADOImageLoader.external.dicomParser = dicomParser;
-
-OHIF.user.getAccessToken = () => {
-  // TODO: Get the Redux store from somewhere else
-  const state = window.store.getState();
-  if (!state.oidc || !state.oidc.user) {
-    return;
-  }
-
-  return state.oidc.user.access_token;
-};
-
-cornerstoneWADOImageLoader.configure({
-  beforeSend: function(xhr) {
-    const headers = OHIF.DICOMWeb.getAuthorizationHeader();
-
-    if (headers.Authorization) {
-      xhr.setRequestHeader('Authorization', headers.Authorization);
-    }
-  },
-});
+// getCornerstoneWADOImageLoader().then(cornerstoneWADOImageLoader => {
```

Comment:
You can delete all of these comments. It was odd from the outset that we initialized in this file.

Reply:
done